### DISCUSSION TITLE
Update mail templates

### DISCUSSION THREAD

**Discussion by leisterfrancisco** [eoscostarica/proton-affiliate#381] [Ideas]
# Improve Mail Templates

The purpose of this discussion is to standardize, improve the emails templates and vary the content.

## Problem

Actually, the non-KYC users that have accepted to receive emails are getting 3 emails but two of those mails are repeated for different moments as it is detailed below.

1. (User triggered) - User apply to Proton Affiliate Program but don't have KYC
2. (Backend triggered) - After the 2/10 of `expiration_days` if the user hasn't got KYC, they will get notified with a mail **(same as before)**
3. (Backend triggered) - Once the total time specified in the `expiration_days` is completed and the user hasn't activated their KYC verification, the service will get that user know with a mail and delete him from the waiting list.

## Current mail templates
The next content is without header and footer.

### Moment 1, 2
```
Regarding Your Proton On-Chain Referral Program Request for ${account}

We were unable to activate your Proton referral link as your account must first complete the KYC (Know Your Customer)  process in the Proton wallet. Once your account has KYC you may apply again and begin using the Proton Referral Program to start earning rewards.

Please, go to the Proton wallet app and submit your KYC application. The KYC process is currently limited to a few countries and plans are underway to increase the list of countries with KYC enabled  To find out what countries can complete KYC please go to the Proton wallet . If you're still having problems after confirming this information, please get in touch with us through Telegram: https://t.me/protonxpr.
```

### Moment 3
```
${account} - Regarding Your Proton On-Chain Referral Program Request

Your Proton referral link is not ready yet. To begin using the Proton Referral Program and start earning rewards, you must first complete the KYC (Know Your Client) authentication process in the Proton wallet.

Please, go to the Proton wallet app and submit your KYC application. Once completed, you can start using your referral link. To find out what countries can complete a KYC, go to the Proton wallet app or ask in the Proton Telegram chat: https://t.me/protonxpr. If you're still having problems after confirming this information, please get in touch with us through Telegram: https://t.me/eoscr.
```

## Visual content

Next are the images of each template for different moments.

### Moment 1, 2
![image](https://user-images.githubusercontent.com/28828796/151888692-c54e3882-7a9f-4cc7-b988-5210a1071994.png)

### Moment 3
![image](https://user-images.githubusercontent.com/28828796/151888653-be76dd23-4a4c-472c-91ea-be3aac2b2be3.png)

**Comment by xavier506** [comment]
### Email 1 

Regarding the Proton Affiliate Program for Your Account ${account}

We were unable to activate your Proton referral link as your account must first complete the KYC (Know Your Customer) process using the Proton wallet. Once your account has completed KYC you may apply again to begin using the Proton Referral Program to start earning rewards.

Please go to the Proton wallet app and submit your KYC application. The KYC process is currently limited to a few countries and plans are underway to increase the list of countries with KYC enabled  To find out what countries can complete KYC please go to the Proton wallet . If you're still having problems after confirming this information, please get in touch with us through Telegram: https://t.me/protonxpr.

**Comment by xavier506** [comment]
### Email 2
Regarding the Proton Affiliate Program for Your Account ${account}

Your Proton referral link is not available yet. To begin using the Proton Referral Program and start earning rewards, you must first complete the KYC (Know Your Client) verification process using the Proton wallet.

Please go to the Proton wallet app and submit your KYC application. Once completed, you will obtain your referral link. To find out what countries can complete a KYC, go to the Proton wallet app or ask in the Proton Telegram chat: https://t.me/protonxpr.

**Comment by ldrojas** [comment]
@leisterfrancisco @xavier506 

**1,2:**

Regarding Your Proton On-Chain Referral Program Request for ${account}

We were unable to activate your Proton referral link. First, you must complete the KYC (Know Your Customer) process for your account in the Proton wallet. Once completed, you may apply again and begin using the Proton Referral Program to earn rewards.

Visit the Proton wallet app to submit your KYC application. The KYC process is limited to a few countries, and plans are underway to expand this list. Go to the Proton wallet to find out if your country is included. If you're still having problems after confirming this information, please contact us through Telegram: https://t.me/protonxpr.



**3:**

${account} - Regarding Your Proton On-Chain Referral Program Request

Your Proton referral link isn’t ready yet. You must first complete the KYC (Know Your Client) authentication process in the Proton wallet to begin using the Proton Referral Program and earn rewards.

Visit the Proton wallet app to submit your KYC application. Once completed, you can start using your referral link. The KYC process is currently limited to a few countries. Go to the Proton wallet to find out if your country is included, or ask in the Proton Telegram chat: https://t.me/protonxpr. If you're still having issues, please contact us through Telegram: https://t.me/eoscr.
